documentation/src/main/markdown/packagemgrs/gradle.md

```diff
@@ -46,4 +46,8 @@ The buildless gradle detector uses Project Inspector to find dependencies and do
 It currently supports "build.gradle" and does not support Kotlin build files.
 
 As of [company_name] [solution_name] 9.5.0 the version of Project Inspector in use supports the `--build-system GRADLE` argument in place of `--strategy GRADLE`.
-The `--force-gradle-repos "url"` argument will be removed from support in the next [company_name] [solution_name] major release and replaced with the `--conf "maven.repo:url"` argument.
\ No newline at end of file
+The `--force-gradle-repos "url"` argument will be removed from support in the next [company_name] [solution_name] major release and replaced with the `--conf "maven.repo:url"` argument.
+
+### Rich version declaration support
```

Comment:
Because this isn't supported by Project Inspector (buildless), can this section be moved above the "Gradle detector buildess" section to avoid confusion.

Reply:
Good idea. Thank you.